pages/en/android/camera-orientation.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Camera orientations
+metadesc: An in-depth guide on how to handle orientations in Android camera apps.
+date: 2020-09-23
+weight: 1
+tags:
+  - camera orientation
+  - large screens
```

Comment:
```suggestion
  - camera
```

Please remove the "large screens" tag as this article doesn't appear to be about utilizing extra space for larger screens. If that's not correct, feel free to keep it. Camera in next comment